common/help-on-spot-models/src/ormConfig.ts

```diff
@@ -1,8 +1,8 @@
  export const ormConfig = {
   "type": "postgres",
   "host": process.env.DATABASE_HOST,
-  "port": 5432,
-  "username": process.env.DATABASE_USERNAME,
-  "password": process.env.DATABASE_PASSWORD,
-  "database": process.env.DATABASE_NAME
+  "port": 6000,
+  "username": "postgres",
+  "password": "postgres",
+  "database": "postgres"
```

Comment:
Please don't change the ormConfig, create a `.env` file instead.

Reply:
Oops. Shouldn't have have bin commited.